cloudknot/cloudknot.py

```diff
@@ -824,7 +819,7 @@ def __init__(
 
         bid_percentage : int, optional
             Compute environment bid percentage if using spot instances
```

Comment:
Even though the data type is specified, I would add this to avoid confusion:

```suggestion
            Compute environment bid percentage if using spot instances (integer between 0 and 100)
```